content/ai/concepts/generative-ai/concepts/code/code.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Code'
+Description: 'Coding was one of the first areas commercialized with GenAI.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
```

Comment:
```suggestion
```
Not on the [subjects list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).